src/_pages/en/index.ejs

```diff
@@ -1,10 +1,75 @@
 ---
 layout: default.ejs
-title: Home page
+title: Radzima.app
 ---
 
+
 <div class="container">
-    <h1>Travel and discover Belarus you didn't know before</h1>
-    <p>Discover national parks, historical sites and attractions with the guide app. Look for excursions and routes in the catalog, and go on an exciting journey.</p>
-    <p><a href="/sitemap"><%= lang[locale].sitemap %></a></p>
+    <div class="c-promo">
+        <div class="c-promo__body">
+            <h1>Travel. Explore. Seize the moment. Discover new unknown Belarus.</h1>
+            <p>Stunning places, unique nature, sightings and excursions, unusual routes - choose whatever you want and go towards new impressions.</p>
+            <div class="c-promo__btns">
+                <a href="<%= site.appGooglePlay %>" target="_blank" rel="noopener">
+                    <img alt="Get it on Google Play" src="/images/app-google-play.svg" width="167" height="48">
+                </a>
+                <a href="<%= site.appAppleStore %>" target="_blank" rel="noopener">
+                    <img alt="Download on the App Store" src="/images/app-app-store.svg" width="166" height="48">
+                </a>
+            </div>
+        </div>
+        <div class="c-promo__img">
+            <img src="/images/promo.svg" alt="">
```

Comment:
What do you think of putting alt here? Is it [decorative or informative](https://weblind.ru/inner.html#image-content)?

Reply:
It can go as it is, absolutely no reason to use it as background. Empty alt is not forbidden, just case when bots/readers should not care what exactly is on the image